src/tools/pmempool/create.c

```diff
@@ -535,6 +535,13 @@ pmempool_create_func(const char *appname, int argc, char *argv[])
 				return -1;
 			}
 		}
+
+		if (PMEM_POOL_TYPE_OBJ == pc.params.type) {
+			if (pc.layout != NULL) {
+				memcpy(pc.params.obj.layout, pc.layout,
+					sizeof(pc.params.obj.layout));
```

Comment:
I think you may read uninitialized value here - pc.layout comes from getopt and may be shorter than sizeof(pc.params.obj.layout)